jekyll/_cci2/creating-orbs.md

```diff
@@ -2,239 +2,140 @@
 
-For a full list of help commands inside the CLI, visit the [CircleCI CLI help](https://circleci-public.github.io/circleci-cli/circleci_orb.html).
+### How It Works
 
-## Creating a CircleCI Orb
+If you want to dive deeper into the Orb Development Kit and get a look at how the components work together to publish your orb, this section is for you.
 
-This section describes each step of the orb publishing process so you will have a better understanding of how to publish your own orb. These examples enables you to follow the process step-by-step to ensure you write an orb that both adheres to CircleCI requirements while also meeting your own needs.
+The [circleci orb init]({{site.baseurl}}/2.0/orb-author/#getting-started) command is responsible for cloning a customizing an [orb template repository](https://github.com/CircleCI-Public/Orb-Project-Template) for your orb, including a pre-written CircleCI configuration file designed with our optimal orb development pipeline.
 
-The following sections describe each step in the orb authoring and publishing process:
+Included in the [/.circleci](https://github.com/CircleCI-Public/Orb-Project-Template/tree/master/.circleci) directory is a README with a breakdown of the included workflows.
 
-### CircleCI Settings
+Your orb pipeline occurs in two workflows:
+* [test-pack](#test-pack)
+* [integration-test_deploy](#integration-test_deploy)
 
-In the CircleCI app Settings page for your project, pipelines must be enabled (default is to be ON for all new projects). The organization owner must also opt-in to use of uncertified orbs in your organization under the Settings tab on the Security page of the CircleCI app.
+#### test-pack
 
-### Get the new CircleCI CLI
+This workflow is the first of two workflows to run, and will execute any time code is committed to the repository for any branch. You can view [this workflow in the template project here](https://github.com/CircleCI-Public/Orb-Project-Template/blob/0354adde8405564ee7fc77e21335090a080daebf/.circleci/config.yml#L40).
 
-The CircleCI platform enables you to write orbs using the CircleCI CLI. If you choose to work with the CLI, the process of writing orbs will be more efficient because you will be able to use existing CircleCI CLI tools and commands.
+The `test-pack` workflow is responsible for all testing we can accomplish up until a development version of an orb is to be published. The reason this separation exists is, to create a development version of an orb, the [orb-tools/publish-dev](https://github.com/CircleCI-Public/Orb-Project-Template/blob/0354adde8405564ee7fc77e21335090a080daebf/.circleci/config.yml#L62) job requires access to our Personal Access Token which will be secured in a [Restricted Context]({{site.baseurl}}/2.0/contexts/#restricting-a-context). Using a Restricted Context allows us to store our token as an environment variable, but only when the job has been triggered by us, or anyone else with access to the context.
```

Comment:
I can't quite get my head round what you mean when you say "the reason this separation exists is...." what separation?

Reply:
@rosieyohannan So, this whole "pipeline" happens in two workflows, which is strange. On CircleCI you would never do what we do here, we have one workflow trigger the next one with the API. We do this as kind of a "hack" or technical limitation.

The first Workflow builds the orb, but we cant immediately test it, because at the time our config was parsed by CircleCI, the orb did not exist. So we have to kick off a new workflow to test the orb we just built.

Would love to make this clearer if you have any thoughts.

Reply:
Ah right I see now. Yeah I will try to make it clearer if I can